src/views/lighttable.c

```diff
@@ -2439,6 +2460,74 @@ void gui_init(dt_view_t *self)
   g_signal_connect(G_OBJECT(display_profile), "value-changed", G_CALLBACK(display_profile_callback), NULL);
 }
 
+static void _dnd_get_picture_reorder(GtkWidget *widget, GdkDragContext *context, gint x, gint y,
+                                    GtkSelectionData *selection_data, guint target_type, guint time,
+                                    gpointer data)
+{
+  GList *selected_images = dt_collection_get_selected(darktable.collection, -1);
+  const int32_t mouse_over_id = dt_control_get_mouse_over_id();
+  dt_collection_move_before(mouse_over_id, selected_images);
+
+  dt_control_button_released(x, y, GDK_BUTTON1_MASK, 0 & 0xf);
```

Comment:
What is this for?

Reply:
That is the code for inserting an image into the right place